src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionFormatsTest.scala

```diff
@@ -17,10 +18,10 @@ class V2AppDefinitionFormatsTest
   import Formats.PathIdFormat
 
   object Fixture {
-    val a1 = V2AppDefinition(
+    val a1 = AppDefinition(
       id = "app1".toPath,
       cmd = Some("sleep 10"),
-      version = Timestamp(1)
+      versionInfo = AppDefinition.VersionInfo.OnlyVersion(Timestamp(1))
```

Comment:
There is no version in AppDefinition compared to V2AppDefinition. Is that the correct equivalent?